p2p/transport/tcp/metrics.go

```diff
@@ -1,4 +1,4 @@
-//go:build !windows
+//go:build !windows && !riscv64
```

Comment:
```suggestion
// riscv64 see: https://github.com/marten-seemann/tcp/pull/1

//go:build !windows && !riscv64
```

Reply:
Are you sure this works? Doesn't the `go:build` need to come first?

Reply:
I don't think so, `go:build` line has to be come before `package`:
```console
> cat a.go 
// test

//go:build doNotBuild

package aaa

const "test" + 42
> cat b.go 
package aaa

func A() {}
> go build .
> echo $status
0
```